filebeat/module/postgresql/log/_meta/fields.yml

```
@@ -28,25 +43,74 @@
       description: >
         Name given to a query when using extended query protocol. If it is "<unnamed>", or not present,
         this field is ignored.
+    - name: command_tag
+      example: "SELECT"
+      description: >
+        Type of session's current command.
+        The complete list can be found at: src/include/tcop/cmdtaglist.h
+    - name: session_start_time
+      type: date
+      description: Time when this session started.
+    - name: virtual_transaction_id
+      description: Backend local transaction id .
+    - name: transaction_id
+      type: long
+      description: The id of current transaction.
+    - name: sql_state_code
+      description: >
+        State code returned by Postgres (if any).
+        This code is not a number.
+        See also https://www.postgresql.org/docs/current/errcodes-appendix.html
```

Nit. Perhaps it is not needed to show in the public docs that this is not a number, reference to postgres docs is probably enough, we can keep the sentence in a comment here so this is not changed to a numeric type.
```suggestion
    - name: sql_state_code
      # This code is not a number.
      type: keyword
      description: >
        State code returned by Postgres (if any).
        See also https://www.postgresql.org/docs/current/errcodes-appendix.html
```